distributed/http/scheduler/prometheus/core.py

```diff
@@ -49,9 +50,15 @@ def collect(self) -> Iterator[GaugeMetricFamily | CounterMetricFamily]:
             - len(self.server.saturated),
         )
         worker_states.add_metric(["saturated"], len(self.server.saturated))
+        paused_workers = len(
+            [w for w in self.server.workers.values() if w.status == Status.paused]
+        )
```

Comment:
I suspect looping over this once every poll should be fine, we can address this later if it's found to become a problem with very large clusters.